packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -110,15 +124,45 @@ class GameWidget<T extends Game> extends StatefulWidget {
     this.focusNode,
     this.autofocus = true,
     MouseCursor? mouseCursor,
-  }) : super(key: key) {
+  })  : create = null,
+        super(key: key) {
     if (mouseCursor != null) {
-      game.mouseCursor = mouseCursor;
+      game!.mouseCursor = mouseCursor;
     }
     if (initialActiveOverlays != null) {
-      game.overlays.addAll(initialActiveOverlays);
+      game!.overlays.addAll(initialActiveOverlays);
     }
   }
 
+  /// Renders a [Game] in a flutter widget tree.
+  ///
+  /// Unlike the default constructor [GameWidget.new], this creates a
+  /// [GameWidget] that controls the creation of the game instance, r
+  /// removing the necessity of rendering this widget inside a [StatefulWidget].
+  ///
+  /// Ex:
+  /// ```
+  /// ...
+  /// Widget build(BuildContext  context) {
+  ///   return GameWidget.controlled(
+  ///     create: MyGameClass.new,
+  ///   )
+  /// }
+  /// ...
+  /// ```
+  const GameWidget.controlled({
+    Key? key,
+    required GameCreate<T> this.create,
```

Comment:
If the type is renamed to `GameBuilder`, rename this to the same noun.

```suggestion
    required GameCreate<T> this.builder,
```

Reply:
Ditto